content/docs/extensions/model/sklearn.md

```diff
@@ -0,0 +1,69 @@
+# Scikit-Learn Models Support
+
+[ModelType](/doc/user-guide/mlem-abcs#modeltype) implementations for any
+sklearn-compatible classes as well as `Pipeline`
+
+## Description
+
+**TODO**
+
+## Requirements
+
+```bash
+pip install mlem[sklearn]
+# or
+pip install scikit-learn
+```
+
+## Examples
+
+### Saving and loading Scikit-Learn model
```

Comment:
Better to use `## Example: Title` if you want them to get listed in the right-hand ToC automatically:

<img width="1088" alt="image" src="https://user-images.githubusercontent.com/1477535/191442211-4d218b23-36f9-4af7-97de-c4d20e1b352b.png">